quoridorui/src/main/java/com/harush/zitoon/quoridor/ui/view/MainMenu.java

```diff
@@ -23,6 +23,11 @@ public void start(Stage primaryStage){
 			Parent root = FXMLLoader.load(getClass().getResource("/resources/layouts/mainmenu.fxml"));
 			Scene scene = new Scene(root);
 			primaryStage.setTitle("Quoridor");
+
+			//theres a method of LoadScrean, why dont we use it?
```

Comment:
Good question, I'm not sure but I think it might be doing something slightly different. 
You can try and if it's better replace this code with a call to `loadScreen()`

Reply:
i think ill skip it